bin/list-nginx-class-ingresses.rb

```diff
@@ -15,7 +15,9 @@ def main
   no_annotation_ingresses = list_ingresses_no_annotations
   list = nginx_class_ingresses + null_class_ingresses + no_annotation_ingresses
 
-  File.write(NGINX_CLASS_INGRESS_LIST_FILE, list.to_json)
+  puts "Total ingress listed "
```

Comment:
Did you want these to be two separate lines of output? If not, you could do this:
```
puts "Total ingress listed: #{list.size}"
```